test/cases/load_test.py

```diff
@@ -0,0 +1,80 @@
+import os
+import json
+from locust import HttpUser, task, tag, events
+
+FAIL_RATIO = float(os.getenv("LT_FAIL_RATIO", "0.01"))
+MEDIAN_RESPONSE_TIME = int(os.getenv("LT_MEDIAN_RESPONSE_TIME", "280"))
+PERCENTILE_95_RESPONSE_TIME = int(os.getenv("LT_PERCENTILE_95_RESPONSE_TIME",
+    "500"))
```

Comment:
I guess these should match the SLIs defined in our app-sop. I'm not opposed to tweaking the SLIs a little, but currently the sli is 'average <= 200ms', so we should at least test that one as well.

Reply:
OOps, I didn't followed the SLIs to set the threshold, I just observed and put a threshold a bit above the current values.
Do the SLI mention the requests per seconds though ? I think the combination of both should be verified. This is because the response time per request changes drastically under a load of 50 requests / s and a load of 80 requests /s